### PR TITLE
haskell: add helm-hoogle, bind to mhf

### PR DESCRIPTION
**Pull request by jb55**

### CODE REVIEW

File: layers/+lang/haskell/packages.el [hunk 1]
```diff
@@ -28,6 +29,10 @@
   (use-package cmm-mode
     :defer t))
 
+(defun haskell/init-helm-hoogle ()
```
Comment (d12frosted) [inline layers/+lang/haskell/packages.el]: I would also check that `helm` is used.

Comment (jb55) [inline layers/+lang/haskell/packages.el]: done :+1:

Comment (jb55) [inline layers/+lang/haskell/packages.el]: looks like I broke something, fixing...

**Comment by joehillen** [comment]
I want this!

**Comment by jb55** [comment]
I fudged and deleted the branch by accident, new PR is here https://github.com/syl20bnr/spacemacs/pull/4625

**Pull request closed without merging**